notebooks/source/model_rendering.ipynb

```diff
@@ -0,0 +1,558 @@
+       "<text text-anchor=\"middle\" x=\"76\" y=\"-15.8\" font-family=\"Times,serif\" font-size=\"14.00\">N</text>\n",
+       "</g>\n",
+       "<!-- m -->\n",
+       "<g id=\"node1\" class=\"node\">\n",
+       "<title>m</title>\n",
+       "<ellipse fill=\"white\" stroke=\"black\" cx=\"54\" cy=\"-201\" rx=\"27\" ry=\"18\"/>\n",
+       "<text text-anchor=\"middle\" x=\"54\" y=\"-197.3\" font-family=\"Times,serif\" font-size=\"14.00\">m</text>\n",
+       "</g>\n",
+       "<!-- sd -->\n",
+       "<g id=\"node2\" class=\"node\">\n",
+       "<title>sd</title>\n",
+       "<ellipse fill=\"white\" stroke=\"black\" cx=\"27\" cy=\"-129\" rx=\"27\" ry=\"18\"/>\n",
+       "<text text-anchor=\"middle\" x=\"27\" y=\"-125.3\" font-family=\"Times,serif\" font-size=\"14.00\">sd</text>\n",
+       "</g>\n",
+       "<!-- m&#45;&gt;sd -->\n",
+       "<g id=\"edge1\" class=\"edge\">\n",
+       "<title>m&#45;&gt;sd</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M47.6,-183.41C44.49,-175.34 40.67,-165.43 37.17,-156.35\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"40.4,-155.03 33.54,-146.96 33.87,-157.55 40.4,-155.03\"/>\n",
+       "</g>\n",
+       "<!-- obs -->\n",
+       "<g id=\"node3\" class=\"node\">\n",
+       "<title>obs</title>\n",
+       "<ellipse fill=\"grey\" stroke=\"black\" cx=\"54\" cy=\"-57\" rx=\"27\" ry=\"18\"/>\n",
+       "<text text-anchor=\"middle\" x=\"54\" y=\"-53.3\" font-family=\"Times,serif\" font-size=\"14.00\">obs</text>\n",
+       "</g>\n",
+       "<!-- m&#45;&gt;obs -->\n",
+       "<g id=\"edge2\" class=\"edge\">\n",
+       "<title>m&#45;&gt;obs</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M57.65,-182.91C59.68,-172.57 61.98,-159.09 63,-147 64.34,-131.06 64.34,-126.94 63,-111 62.28,-102.5 60.93,-93.31 59.49,-85.01\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"62.91,-84.29 57.65,-75.09 56.03,-85.56 62.91,-84.29\"/>\n",
+       "</g>\n",
+       "<!-- sd&#45;&gt;obs -->\n",
+       "<g id=\"edge3\" class=\"edge\">\n",
+       "<title>sd&#45;&gt;obs</title>\n",
+       "<path fill=\"none\" stroke=\"black\" d=\"M33.4,-111.41C36.51,-103.34 40.33,-93.43 43.83,-84.35\"/>\n",
+       "<polygon fill=\"black\" stroke=\"black\" points=\"47.13,-85.55 47.46,-74.96 40.6,-83.03 47.13,-85.55\"/>\n",
+       "</g>\n",
+       "</g>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<graphviz.dot.Digraph at 0x1072507c0>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data = jnp.ones(10)\n",
+    "numpyro.render_model(model, model_args=(data,))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "false-qualification",
+   "metadata": {},
+   "source": [
+    "The visualization can be saved to a file by providing `filename='path'` to `numpyro.render_model`. You can use different formats such as PDF or PNG by changing the filename's suffix."
```

Comment:
Could you also mention that if `filename` is not specified, we can also change the format by using e.g. `graph.format = "png"`